bootstrap.sh

```
@@ -1 +1,23 @@
 #!/bin/bash
+
+echo "Creating Kubernetes cluster using kind with extra Mount"
+kind create cluster --config cluster.yml
```

This runs `kind create cluster --config cluster.yml` which is correct if `cluster.yml` exists at the script’s working directory. Ensure `cluster.yml` is indeed at repository root (or update the path). Also, create a small wait after creating the cluster so `kubectl` operations won’t fail when the cluster is not yet ready. Example: `kubectl wait --for=condition=Ready nodes --all --timeout=120s` or poll `kubectl get nodes` before proceeding. Consider adding `set -euo pipefail` at the top so the script fails fast on errors (robustness requirement).